.github/workflows/default.yml

```diff
@@ -44,7 +43,7 @@ jobs:
           ls -l /opt/hostedtoolcache/Python/3.9.1/x64/bin || true
           echo "/opt/hostedtoolcache/Python/3.9.1/x64/bin" >> $GITHUB_PATH
           ansible --version
-          cd $GITHUB_WORKSPACE/${{ env.ROLEPATH }}
+          cd $GITHUB_WORKSPACE/ebba.bigbluebutton
```

Comment:
I guess the reason here could also be the typo
```diff
-ebba
+ebbba
```

Reply:
yes, typo. but consistent with checkout path so unlikely to be the root cause.
fixed.